rclrs/build.rs

```diff
@@ -63,9 +29,26 @@ fn main() {
 
     if let Ok(ament_prefix_paths) = ament_prefix_var {
         for ament_prefix_path in ament_prefix_paths.split(':') {
-            if let Some(clang_args) = generate_clang_include_arg(ament_prefix_path) {
-                builder = builder.clang_args(clang_args);
+            // Should not panic, since this was originally a valid UTF-8 String
+            builder = builder.clang_arg(format!("-I{}/include", ament_prefix_path));
+
+            // If this is RCL or rcutils, we need to add a special include
```

Comment:
Can we simply do
```
for pkg in &["rcl", "rcutils", etc] {
    builder = builder.clang_arg(format!("-I{}/include/{}", ament_prefix_path, pkg))
}
```
? I'm still not convinced that this version, where the package name is assumed to be the last element of `AMENT_PREFIX_PATH`, will work.

Reply:
If I do that, we'll have two for loops, one after the other. May not be a huge issue, but still.

When I built and ran this using `--merge-install` everything seemed to work fine. `AMENT_PREFIX_PATHS` seems to hold to the rule that the package name is on the end of each of the paths, though the paths are now in the `ros2_rolling/install` folder.

Reply:
Can you tell me what's in your `AMENT_PREFIX_PATH` when you build `rclrs`, and the output of `ls` for each of these directories?

Reply:
Not sure if you saw the comment in the chat – I verified that the nested loop is necessary and that it doesn't build when the `rcl` etc. packages are part of a `--merge-install` install. So I propose to add the nested loop.

A possible performance optimization would be to add only those paths that exist.